packages/next-app/src/app/internalmanager/(components)/TokenTable.tsx

```diff
@@ -1,14 +1,19 @@
 "use client";
 import { InternalBalanceQuery } from "@balancer-pool-metadata/gql/src/balancer-internal-manager/__generated__/Mainnet";
 import { networkFor } from "@balancer-pool-metadata/shared";
+import {
+  MinusCircledIcon,
+  PlusCircledIcon,
+  WidthIcon,
```

Comment:
there's not a circled option for this icon, right? could you style it to look like the others?